fix(apple): Clarify app start tracing (#6789)
getsentry/sentry-docs: src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx

This feature is available for iOS, tvOS, and Mac Catalyst.

App Start Tracing is enabled by default once you <PlatformLink to="/performance/">set up performance monitoring</PlatformLink>. This feature provides insight into how long your application takes to launch. It adds spans for different phases, from the application launch to the first auto-generated UI transaction.

> The SDK adds the spans to the first started transaction during the app launch with the operation `ui.load`, which the UIViewController tracing uses. Suppose the time interval between the transaction start time and the end timestamp of the measured app start is more significant than 5 seconds. In that case, the SDK doesn't attach the app start spans to avoid attaching it to possibly unrelated transactions.

Reviewer: ```suggestion

The SDK adds these spans to the first transaction that starts during the app launch with the operation `ui.load`. This is what UIViewController tracing uses. If the duration between the transaction start time and the end timestamp of the app start is more than 5 seconds, the SDK doesn't attach the spans from the app start. This helps prevent attaching spans to unrelated transactions.
```

Make this a separate paragraph and tweak phrasing for clarity.
